cli/internal/prune/prune.go

```diff
@@ -281,6 +281,15 @@ func (p *prune) prune(opts *turbostate.PrunePayload) error {
 			); err != nil {
 				return errors.Wrap(err, "Failed copying patch file")
 			}
+			if opts.Docker {
```

Comment:
This appears to copy to both the full dir and the json dir, while the original suggestion was to copy to json instead of full. Is that on purpose?

Reply:
Yes, json should just be a subset of full. `json` is just what is needed to install and `full` is what is needed to use the scoped workspace